### ISSUE TITLE
Unable to install tideways.

### ISSUE THREAD

**Issue by fubarhouse** [geerlingguy/drupal-vm#1654]
### Issue Type

Bug Report / Support Request

### Your Environment

```
Vagrant 2.0.1
VirtualBox 5.1.22r115126
ansible 2.4.0.0
  config file = None
  configured module search path = [u'/Users/education/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python2.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 2.7.14 (default, Sep 25 2017, 09:54:19) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.37)]
```

### Your OS

**macOS** (10.13.2)

### Full console output

As described in https://github.com/geerlingguy/ansible-role-php-tideways/issues/3, the tideways role is failing. @oxyc has done some preliminary work to help however in its current state appears to be unusable.

### Summary

In attempting to install tideways, to try it out and see how it differs from xhprof, I came across the same fatal error raised in the role just over a week ago. For now, I'll just keep using xhprof as it does seem to work fine with PHP 7.1.

**Comment by geerlingguy** [comment]
And the error message, for posterity:

```
TASK [geerlingguy.php-tideways : Build Tideways.] ******************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: OSError: [Errno 2] No such file or directory: '/root/php7.1/php-profiler-extension-master'
failed: [drupalvm] (item=phpize) => {"changed": false, "item": "phpize", "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/ansible__1HGJs/ansible_module_command.py\", line 213, in <module>\n    main()\n  File \"/tmp/ansible__1HGJs/ansible_module_command.py\", line 152, in main\n    os.chdir(chdir)\nOSError: [Errno 2] No such file or directory: '/root/php7.1/php-profiler-extension-master'\n", "module_stdout": "", "msg": "MODULE FAILURE", "rc": 1}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: OSError: [Errno 2] No such file or directory: '/root/php7.1/php-profiler-extension-master'
failed: [drupalvm] (item=./configure) => {"changed": false, "item": "./configure", "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/ansible_QmdphF/ansible_module_command.py\", line 213, in <module>\n    main()\n  File \"/tmp/ansible_QmdphF/ansible_module_command.py\", line 152, in main\n    os.chdir(chdir)\nOSError: [Errno 2] No such file or directory: '/root/php7.1/php-profiler-extension-master'\n", "module_stdout": "", "msg": "MODULE FAILURE", "rc": 1}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: OSError: [Errno 2] No such file or directory: '/root/php7.1/php-profiler-extension-master'
failed: [drupalvm] (item=make) => {"changed": false, "item": "make", "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/ansible_rEd419/ansible_module_command.py\", line 213, in <module>\n    main()\n  File \"/tmp/ansible_rEd419/ansible_module_command.py\", line 152, in main\n    os.chdir(chdir)\nOSError: [Errno 2] No such file or directory: '/root/php7.1/php-profiler-extension-master'\n", "module_stdout": "", "msg": "MODULE FAILURE", "rc": 1}
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: OSError: [Errno 2] No such file or directory: '/root/php7.1/php-profiler-extension-master'
failed: [drupalvm] (item=make install) => {"changed": false, "item": "make install", "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/ansible_M8_eWL/ansible_module_command.py\", line 213, in <module>\n    main()\n  File \"/tmp/ansible_M8_eWL/ansible_module_command.py\", line 152, in main\n    os.chdir(chdir)\nOSError: [Errno 2] No such file or directory: '/root/php7.1/php-profiler-extension-master'\n", "module_stdout": "", "msg": "MODULE FAILURE", "rc": 1}
```

This is using all the Drupal VM defaults, just adding `tideways` to the list of `installed_extras` (as is indicated in the docs).

**Comment by geerlingguy** [comment]
@oxyc I merged your PR (which was tested/working!) and now it's broken :(

I will try to take a look at what's going on later today, but might not get time.

**Comment by geerlingguy** [comment]
Looks like they went back to https://github.com/tideways/php-xhprof-extension from php-profiler-extension 😑

**Comment by geerlingguy** [comment]
Fixed in the 2.0.1 role version. Testing now.
